### ISSUE TITLE
How do you mask the implicit knowledge

### ISSUE THREAD

**Issue by PamelaDDD** [guoyang9/UnifER#5]
Hello scholar, you mentioned in the paper that the model parameters will be randomly initialized to mask the implicit knowledge. Does this mean that the pre-training weights are not loaded? But I don't load the pre-training weights, and the score is not as high as Figure 4 in the paper. How did I achieve it?

**Comment by guoyang9** [comment]
The green bars from the right half of Figure 4 show the results without loading pre-trained weights.
The performance of these models is already quite low, are you sure your scores are lower than these>

**Comment by PamelaDDD** [comment]
when I load vilt in this way （without loading pre-trained weights） and I get the score well below 30, which is shown in Fig 4.&nbsp;Is the way I understand it wrong, or will additional parameters be adjusted






Pamela
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "guoyang9/UnifER"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年1月11日(星期三) 中午1:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [guoyang9/UnifER] How do you mask the implicit knowledge (Issue #5)





 
The green bars from the right half of Figure 4 show the results without loading pre-trained weights.
 The performance of these models is already quite low, are you sure your scores are lower than these&gt;

**Comment by guoyang9** [comment]
If you use the same setting as me, the accuracy will be around 32%, but a small variation (e.g., 2 points) is also possible when the experimental settings are different.

**Comment by PamelaDDD** [comment]
Is my initialization code correct? In fact, I used the same settings, but the accuracy was only about 10%


Pamela
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "guoyang9/UnifER"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年1月11日(星期三) 下午2:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [guoyang9/UnifER] How do you mask the implicit knowledge (Issue #5)





 
If you use the same setting as me, the accuracy will be around 32%, but a small variation (e.g., 2 points) is also possible when the experimental settings are different.

**Comment by guoyang9** [comment]
I forget the details.
But if you want to use randomly initialized parameters, you should change this [line](https://github.com/guoyang9/UnifER/blob/3b543f8c57214aaadbc45573bb56e95086c24152/modules/model_build.py#L21) to `model = eval(config['transformer']['vlp_model'] )`.

Hope this helps.
